app/presenters/outcome_presenter.rb

```diff
@@ -19,6 +19,18 @@ def title
     @renderer.content_for(:title)
   end
 
+  def heading_1_title
+    title_as_heading? ? title : @flow_presenter.title
+  end
+
+  def heading_2_title
+    title_as_heading? ? @flow_presenter.title : title
+  end
```

Comment:
Ah sorry, we don't want to display the flow title at all if `title_as_heading?` is true. I.e. heading_2_title = nil

Reply:
Fixed.